AC-Music.py

```diff
@@ -1,81 +1,10 @@
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '02 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '03 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '04 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '05 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '06 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '07 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '08 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '09 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '10 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '11 AM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '12 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '01 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '02 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '03 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '04 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '05 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '06 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '07 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '08 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '09 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '10 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
-        elif time_hr == '11 PM' :
-                playsound(mp3_to_play)
-                time_hr = currentDT.strftime("%I %p")
+        mp3_to_play = os.path.join(directory, time_hr + '.mp3')
```

Comment:
We use `os.path.join` to build a string with the right `/` and `\` path separators, as well as (hopefully) dealing with your spaces correctly.